q2_minimap2/trim_long_reads.py

```diff
@@ -0,0 +1,94 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2024, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+import os
+import subprocess
+
+from q2_types.per_sample_sequences import CasavaOneEightSingleLanePerSampleDirFmt
+
+from q2_minimap2._utils import run_commands_with_pipe
+
+
+# Generates a command list for the 'chopper' tool
+# incorporating provided parameters for sequence quality trimming
+def construct_chopper_command(
+    quality: int,
+    maxqual: int,
+    minlength: int,
+    maxlength: int,
+    headcrop: int,
+    tailcrop: int,
+    threads: int,
+) -> list:
+    return [
+        "chopper",
+        "--quality",
+        str(quality),
+        "--maxqual",
+        str(maxqual),
+        "--minlength",
+        str(minlength),
+        "--maxlength",
+        str(maxlength),
+        "--headcrop",
+        str(headcrop),
+        "--tailcrop",
+        str(tailcrop),
+        "--threads",
+        str(threads),
```

Comment:
```suggestion
        "chopper",
        "--quality", str(quality),
        "--maxqual", str(maxqual),
        "--minlength", str(minlength),
        "--maxlength", str(maxlength),
        "--headcrop", str(headcrop),
        "--tailcrop", str(tailcrop),
        "--threads", str(threads),
```
😜 stupid suggestion but somehow aesthetically pleasing

Reply:
Unfortunately, it is a `black` thing. I don't like it either...